### PR TITLE
Refac: Update write to include new columns in time_serie_points basis data

### PR DESCRIPTION
**Pull request by johevemi**
# Description

## Pull-request quality

- [ ] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202409120820_alter_table__basis_data_internal__time_series_points.sql
Comment (johevemi) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202409120820_alter_table__basis_data_internal__time_series_points.sql:1]: remove before merge.

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202409120823_update_table__basis_data_internal__time_series_points.sql
Comment (johevemi) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202409120823_update_table__basis_data_internal__time_series_points.sql:1]: remove before merge. this will be done in another PR

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202409120824_delete_rows__basis_data_internal__time_series_points.sql
Comment (johevemi) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202409120824_delete_rows__basis_data_internal__time_series_points.sql:1]: remove before merge. this will be done a later PR

File: source/databricks/calculation_engine/package/calculation/calculation_core.py [hunk 1]
```diff
@@ -56,6 +56,14 @@ def execute(
                 )
             )
 
+            if is_wholesale_calculation_type(args.calculation_type):
+                with logging_configuration.start_span("calculation.wholesale.prepare"):
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/calculation/calculation_core.py:60]: This message is being logged twice now.

File: source/databricks/calculation_engine/package/databases/wholesale_basis_data_internal/schemas/time_series_points_schema.py [hunk 2]
```diff
@@ -35,3 +35,18 @@
...
+time_series_points_schema_temp = StructType(
+    [
+        StructField(TableColumnNames.calculation_id, StringType(), False),
+        StructField(TableColumnNames.metering_point_id, StringType(), False),
+        StructField(TableColumnNames.quantity, DecimalType(18, 3), False),
+        StructField(TableColumnNames.quality, StringType(), False),
+        StructField(TableColumnNames.observation_time, TimestampType(), False),
+        StructField(TableColumnNames.metering_point_type, StringType(), False),
+        StructField(TableColumnNames.resolution, StringType(), False),
+        StructField(TableColumnNames.grid_area_code, StringType(), False),
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/databases/wholesale_basis_data_internal/schemas/time_series_points_schema.py:49]: Do we know whether these will ever be NOT NULLABLE?

Comment (johevemi) [reply source/databricks/calculation_engine/package/databases/wholesale_basis_data_internal/schemas/time_series_points_schema.py:49]: Depending on whether we can change the nullability of a column i delta table after a table has been created. It should be NOT NULLABLE but I don't know if we can change it in delt, this was meant as a temporary fix and should be fixed in the remove null rows story I have created.

File: source/databricks/calculation_engine/package/calculation/calculation_core.py (hunk 1 above)
Comment (johevemi) [reply source/databricks/calculation_engine/package/calculation/calculation_core.py:60]: oh right, i will remove it

File: source/databricks/calculation_engine/package/databases/wholesale_basis_data_internal/schemas/time_series_points_schema.py (hunk 2 above)
Comment (johevemi) [reply source/databricks/calculation_engine/package/databases/wholesale_basis_data_internal/schemas/time_series_points_schema.py:49]: it is used in tests to validate schema is correct and because prepared_metering_point_time_series has those 3 columns as nullable false, the test is going to fail on validation.

Comment (johevemi) [reply source/databricks/calculation_engine/package/databases/wholesale_basis_data_internal/schemas/time_series_points_schema.py]: i have moved it to the test in question to make it more clear that it is to be removed when nullability is fixed

**Review by energinet-ajw — APPROVED** [review]
